examples/built-in-integer-subtypes/built_in_integer_subtypes.metatags

```diff
@@ -0,0 +1,2 @@
+description: This BBE demonstrates built-in integer subtypes in Ballerina.
+keywords: ballerina, ballerina by example, bbe, int, int subtypes, signed, unsigned
```

Comment:
Shall we add specific signed and unsigned types in here. Because this files includes keywords for searching purposes
ex:- signed32, unsigned32, signed 16, unsigned16.

Reply:
@praneesha, is there a limit on the number of keywords we can include?

Reply:
@MaryamZi - We don't have a limit but let's not make it a huge list too.

Reply:
Then, I think the current keywords will enough.